upgrade_guide/README.md

```diff
@@ -19,6 +19,17 @@ under the License.
 
 ## Upgrade guide
 
+### NEW VERSION PLACEHOLDER
```

Comment:
@absurdfarce Let me know how you prefer this to be handled - can keep it as an area for other contributors to put their notes, then replace the version placeholder as part of the next release.

Reply:
Your suggestion seems like a pretty reasonable approach to me @aratno .  In the past we'd usually just set the placeholder to whatever we thought the next version would be (knowing full well it might be changed as things moved along) but I have no objection to just leaving a placeholder in the doc.  Part of the release checklist could then become "update the placeholder to the correct version string".